h2/src/test/org/h2/test/scripts/functions/string/base64_encode.sql

```diff
@@ -0,0 +1,28 @@
+-- Copyright 2004-2020 H2 Group. Multiple-Licensed under the MPL 2.0,
+-- and the EPL 1.0 (https://h2database.com/html/license.html).
+-- Initial Developer: H2 Group
+--
+
+CALL '>' || base64_encode(stringtoutf8('')) || '<';
+>> ><
+
+CALL base64_encode(stringtoutf8('A'));
+>> QQ==
+
+CALL base64_encode(stringtoutf8('AB'));
+>> QUI=
+
+CALL base64_encode(stringtoutf8('ABC'));
+>> QUJD
+
+CALL base64_encode(stringtoutf8('ABCD'));
+>> QUJDRA==
+
+CALL base64_encode(stringtoutf8('my string'));
+>> bXkgc3RyaW5n
+
+CALL base64_encode(stringtoutf8('про'));
+>> 0L/RgNC+
+
+CALL base64_encode(stringtoutf8('про'), 'URL');
```

Comment:
Please, use only ASCII characters here. To test larger bytes use the binary string instead (`X'0123456789ABCDEF'`).

All invocations of `STRINGTOUTF8()` here can be removed for simplicity.